src/Language/PureScript/Sugar/TypeClasses.hs

```diff
@@ -317,14 +337,16 @@ typeInstanceDictionaryDeclaration name mn deps className tys decls =
     return (PositionedValue pos com val)
   memberToValue _ _ = internalError "Invalid declaration in type instance definition"
 
-typeClassMemberName :: Declaration -> String
-typeClassMemberName (TypeDeclaration ident _) = runIdent ident
-typeClassMemberName (ValueDeclaration ident _ _ _) = runIdent ident
-typeClassMemberName (PositionedDeclaration _ _ d) = typeClassMemberName d
-typeClassMemberName _ = internalError "typeClassMemberName: Invalid declaration in type class definition"
+typeClassMemberNames :: [Declaration] -> [Ident]
+typeClassMemberNames = map go
+  where
+  go (TypeDeclaration ident _) = ident
+  go (ValueDeclaration ident _ _ _) = ident
+  go (PositionedDeclaration _ _ d) = go d
+  go d = error $ "Invalid declaration in type class definition: " ++ show d
 
-superClassDictionaryNames :: [Constraint] -> [String]
+superClassDictionaryNames :: [Constraint] -> [Ident]
```

Comment:
These are only used for the accessors now, yes? I'd probably just get rid, honestly.